rental.rb

```diff
@@ -1,13 +1,13 @@
 class Rental
-  def initialize(book, person)
-    @date = DateTime.now
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] to make the associations work Getters and Setters for @book and @person instance variables should be present